src/main/java/com/twilio/rest/api/v2010/account/CallReader.java

```diff
@@ -321,20 +285,12 @@ private void addQueryParams(final Request request) {
             request.addQueryParam("Status", status.toString());
         }
 
-        if (absoluteStartTime != null) {
-            request.addQueryParam("StartTime", absoluteStartTime.toString(Request.QUERY_STRING_DATE_TIME_FORMAT));
-        } else if (rangeStartTime != null) {
-            request.addQueryDateTimeRange("StartTime", rangeStartTime);
-        }
-
-        if (absoluteEndTime != null) {
-            request.addQueryParam("EndTime", absoluteEndTime.toString(Request.QUERY_STRING_DATE_TIME_FORMAT));
-        } else if (rangeEndTime != null) {
-            request.addQueryDateTimeRange("EndTime", rangeEndTime);
+        if (startTime != null || endTime != null) {
+            request.addQueryDateTimeRange("Range", startTime, endTime);
```

Comment:
`"Range"` is not correct. Where is this coming from?

Reply:
This was a mistake, should be "EndTime", will fix.